src/components/PositionCard/Sushi.tsx

```diff
@@ -48,7 +49,13 @@ export default function SushiPositionCard({ tokenA, tokenB, liquidityToken, bord
           <AutoRow gap="8px">
             <DoubleCurrencyLogo currency0={currency0} currency1={currency1} size={20} />
             <Text fontWeight={500} fontSize={20}>
-              {!currency0 || !currency1 ? <Dots>Loading</Dots> : `${currency0.symbol}/${currency1.symbol}`}
+              {!currency0 || !currency1 ? (
+                <Dots>
```

Comment:
probably dots should just be the `...` part and part of the translation (might want to remove it from other languages)

Reply:
interetsting.. DOT animates the `...`. That may be hard to localize. Maybe we could just use a spinner or something other iconography. I'll leave as-is for now since there are quite a few instance of Dots.